index.html

```diff
@@ -2532,183 +2532,169 @@ <h1>Servient Implementation</h1>
-            is based on this dual role.
+            Depending on the <a>Protocol Binding</a>, <a>Servients</a> can perform in both server and client role,
+            hence the portmanteau naming.
         </p>
-        <p> The previous sections and 
-            chapter <a href="#sec-wot-architecture"></a> 
-            describes how the WoT
-            building blocks conceptually relate to each other and to the
-            abstract architecture. When implementing these concepts, a
+        <p> The previous section describes how the WoT
+            building blocks conceptually relate to each other and how they correspond to the
+            abstract <a href="#sec-wot-architecture">Web Architecture</a>.
+            When implementing these concepts, a
             more detailed view is necessary that takes certain technical
             aspects into account. The detailed architecture of a <a>Servient</a>
-            implementation using the (optional) <a>WoT Scripting API</a> is
+            implementation using the (optional) <a>WoT Scripting API</a> building block is
             shown in <a href="#architecture-implementation"></a>.
-            This figure shows and implementation that is using the optional 
-            <a>WoT Scripting API</a>. 
-            Scripts are actually executed by a nested Scripting Runtime, 
-            while WoT-specific Servient Implementation aspects are managed
-            by the WoT Runtime. 
-            <a>Consumed Thing</a> and <a>Exposed Thing</a>
-            objects provide abstract interfaces to the Things
-            described in related Thing Descriptions. Scripts can
-            operate on Things using these abstractions by performing
-            abstract Interactions while the details of the Protocol Bindings are
-            managed automatically by the WoT Runtime.
+            Here, the <a>WoT Runtime</a> is also a <a>Scripting Runtime</a> system that
+            in addition to managing the WoT-specific aspects also interprets and executes the application scripts.
         </p>
         <figure id="architecture-implementation">
-            <img src="images/wot-thing-scripting.png"
-                style="width: 640px;" />
-            <figcaption>Implementation of a Servient
-                Using the Optional WoT Scripting API.
-            </figcaption>
+            <img src="images/wot-thing-scripting.png" style="width: 80%;" />
+            <figcaption>Implementation of a Servient Using the Optional WoT Scripting API</figcaption>
         </figure>
-        <p> The role and functionality of each additional module shown
+        <p>
+            <a>Servients</a> with <a>WoT Scripting API</a> support usually run on powerful devices, edge nodes, or in the cloud.
+            There are, however, also <a>Scripting Runtimes</a> for resource-constrained systems.
+            Furthermore, the WoT Architecture does not limit <a>WoT Scripting API</a> definitions to JavaScript.
```

Comment:
This sentence is something contradicting with a sentence in [8.4 WoT Scripting API](https://cdn.staticaly.com/gh/w3c/wot-architecture/mkovatsc-patch-section-8/index.html?env=dev#wot-scripting-api) (l. 2631),
> The WoT Scripting API building block defines an ECMAScript API that ...

I think this sentence can be written as below: 
> Furthermore, the WoT Architecture does not limit _northbound WoT Runtime interface_ definition to ECMAScript.  

(implying existence of Native WoT Runtime and Native WoT API)

What do you think?

Reply:
Good catch. I'm not sure if the term "northbound" is known to all readers. Perhaps just call it "application-facing".

Reply:
I will rephrase, so that "WoT Scripting API" refers to the defined ECMAScript API and "other scripting API" are allowed by the architecture.

Also, I will replace north-/southbound.